docs/data/sql_functions.yml

```diff
@@ -580,18 +614,18 @@ json:
       If the error behavior is omitted, `FALSE ON ERROR` is assumed as the default.
 
       ```
-      // TRUE
```

Comment:
99fd8f313c2114128eb0a6ec7f0c4fb7a31a487f should probably still be merged for 1.14. Do you want to backport the commit when merging this PR or should I raise a separate PR for that hotfix?